File: script/python/increment/fizz_buzz_increment.py
import sys
# import time

if __name__ == '__main__':
    n = int(sys.argv[1])

    fizzbuzz = 0
    fizz = 0
    buzz = 0
    none = 0

    # start = time.time()
    for i in range(1, n + 1):
        if i % 15 == 0:
            fizzbuzz += 1
        elif i % 3 == 0:
            fizz += 1
        elif i % 5 == 0:
            buzz += 1
        else:
            none += 1
    # print("elapsed_time:{0}".format(time.time()) + "[sec]")
    print("fizzbuzz:{0}".format(fizzbuzz))
    print("fizz:{0}".format(fizz))
    print("buzz:{0}".format(buzz))
    print("none:{0}".format(none))

File: script/ruby/increment/fizz_buzz_increment.rb
# frozen_string_literal: true

n = ARGV[0].to_i

fizzbuzz = 0
fizz = 0
buzz = 0
none = 0

# start_time = Time.now
Range.new(1, n).each do |i|
  if (i % 15).zero?
    fizzbuzz += 1
  elsif (i % 3).zero?
    fizz += 1
  elsif (i % 5).zero?
    buzz += 1
  else
    none += 1
  end
end

# p "elapsed_time:#{Time.now - start_time}[sec]"
p "fizzbuzz:#{fizzbuzz}"
p "fizz:#{fizz}"
p "buzz:#{buzz}"
p "none:#{none}"
